text.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>text-utilities</title>
+    <link rel="stylesheet" href="text.css">
+</head>
+<body>
+    <nav class="container">
+        <div id="logo">Icarus-UI</div>
+        <div class="a">
+            <a href="#" class="a1">Home</a>
+            <a  href="#"  class="a1">Docs</a>
+        </div>
+    </nav>
+    <div class="grid">  
+    <aside class="side-bar">
+        <div><a href="#">Installation</a></div>
+        <div><a href="#">Typography</a></div>
+        <h2>Components</h2>
+        <div><a href="#">Alert</a></div>
+        <div><a href="#">Avatar</a></div>
+        <div><a href="#">Badge</a></div>
+        <div><a href="#">Button</a></div>
+        <div><a href="#">Card</a></div>
+        <div><a href="#">Grid</a></div>
+        <div><a href="#">Image</a></div>
+        <div><a href="#">Input</a></div>
+        <div><a href="#">Toast</a></div>
+    </aside>
+    <div class="main">
+        <h1>
+            <span class="main-text">
+                Text utilities
+            </span>
+        </h1>
+            <p>
+                Text utilites include headings, different font weights, font sizes,colored texts.
+            </p>
+            <h2>
+                Headings
+            </h2>
+            <div class="heading mg">
+                <h1>Heading 1</h1>
+                <h2>Heading 2</h2>
+                <h3>Heading 3</h3>
+                <h4>Heading 4</h4>
+                <h5>Heading 5</h5>
+                <h6>Heading 6</h6>
+            </div>
+            <iframe
+            src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=htmlmixed&width=680&ds=true&dsyoff=31px&dsblur=0px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=13.5px&lh=119%25&si=false&es=2x&wm=false&code=%2520%253Cdiv%2520class%253D%2522heading%2520mg%2522%253E%250A%2520%2520%2520%2520%253Ch1%253EHeading%25201%253C%252Fh1%253E%250A%2520%2520%2520%2520%253Ch2%253EHeading%25202%253C%252Fh2%253E%250A%2520%2520%2520%2520%253Ch3%253EHeading%25203%253C%252Fh3%253E%250A%2520%2520%2520%2520%253Ch4%253EHeading%25204%253C%252Fh4%253E%250A%2520%2520%2520%2520%253Ch5%253EHeading%25205%253C%252Fh5%253E%250A%2520%2520%2520%2520%253Ch6%253EHeading%25206%253C%252Fh6%253E%250A%2520%253C%252Fdiv%253E"
+            style="width: 225px; height: 186px; border:0; transform: scale(1); overflow:hidden;"
```

Comment:
avoid using inline styles. you can add it in your CSS file and then use the class name